README.md

```diff
@@ -30,36 +32,36 @@ You'll need the below to use the script to generate the installer image:
 * `libfdt-dev`
 * `cmake`
 
-**If you are not interested in building yourself**, the pre-built files are available [here](https://github.com/dangowrt/linksys-e8450-openwrt-installer/releases).
+**If you are not interested in building yourself**, the pre-built files are available [here](https://github.com/dangowrt/owrt-ubi-installer/releases).
 
-## Installing OpenWrt
 
-* **IMPORTANT: *If* a device running stock 1.1.x firmware rejects the installer image, the recommended work-around is to downgrade the device to version 1.0.x, and then re-attempt uploading the installer image.  See "Downgrading Firmware" instructions below.**
+## Upgrading Stock Firmware
+* **IMPORTANT**: If your router is running FW version **1.1.00.180912 or below**, router must upgrade to **v1.1.01.272918** first which does the switch to the new "signed" before you can upgrade firmware to **1.2.00.360516 "signed" or above**.
```

Comment:
Why? There is no need to upgrade the stock firmware before replacing it.

Reply:
Because now only the "signed" files will be used to install OpenWrt and you must have stock firmware "signed" in the router, and it is better to leave that warning in case someone has the router with firmware version 1.1.00.180912 or below.

Don't compare the changes without first looking at the new readme to see what I mean:
https://github.com/Shortguide/owrt-ubi-installer/tree/main

Reply:
Because now only the "signed" files will be used to install OpenWrt and you must have stock firmware "signed" in the router, and it is better to leave that warning in case someone has the router with firmware version 1.1.00.180912 or below to first flash the firmware v1.1.01.27291 that makes the switch to the new "signed" one and can install a "signed" firmware (**v1.2.00.360516**).

Don't compare the changes without first looking at the new readme to see what I mean:
https://github.com/Shortguide/owrt-ubi-installer/tree/main

Reply:
The problem I see there is that newer versions of the stock firmware behave much more annoying. You can no longer abort the wizard on first start, completing it requires a DHCP server connected to the WAN port, ...
If you happen to get a device with an old version of the stock firmware, you should be thankful and take the easy route of using that to flash the unsigned image.
There is no advantage what-so-ever of first upgrading and then using the signed installer image -- everything is going to be replaced anyway no matter if using signed or unsigned installer image, the end result is the same.

Reply:
Because now only the "signed" files will be used to install OpenWrt and you must have stock firmware "signed" in the router, and it is better to leave that warning in case someone has the router with firmware version 1.1.00.180912 or below to first flash the firmware v1.1.01.27291 that makes the switch to the new "signed" one and now can install a stock "signed" firmware (**v1.2.00.360516** or above).

Don't compare the changes without first looking at the new readme to see what I mean:
- https://github.com/Shortguide/owrt-ubi-installer/tree/main


That is the order that must be followed to convert the router to signed firmware, you said it and Linksys and Belkin also say it:
- https://github.com/dangowrt/owrt-ubi-installer/issues/151#issuecomment-1724524554
- https://downloads.linksys.com/support/assets/releasenotes/E8450_1.2.00.360516_Customer_Release_Note.txt
- https://s3.belkin.com/support/assets/belkin/releasenotes/RT3200_1.2.00.360516_Customer_Release_Notes.txt

Reply:
> The problem I see there is that newer versions of the stock firmware behave much more annoying. You can no longer abort the wizard on first start, completing it requires a DHCP server connected to the WAN port, ...
No problem, and it doesn't even take 30 seconds to complete the wizard and whether you want to accept it or not, now 99% of the routers comes with signed firmware and the current guide is too difficult to understand, it is better that everyone has the same firmware "signed" and so you avoid problems and facilitate the understanding of the guide for everyone.

> If you happen to get a device with an old version of the stock firmware, you should be thankful and take the easy route of using that to flash the unsigned image.
In the old "unsigned" stock firmware the recovery mode does not work (hold down the "reset" button while turning on the device), but in the "signed" firmware the recovery mode works (I'm talking about stock firmware not OpenWrt) and maybe one day it can be useful for someone.

> There is no advantage what-so-ever of first upgrading and then using the signed installer image -- everything is going to be replaced anyway no matter if using signed or unsigned installer image, the end result is the same.
These changes to the guide will make it easier for anyone to understand, because right now it has too much useless and repetitive information and prevents you from being asked why I can't install the firmware or why I can't downgrade the firmware or why my router bricked when I was doing the full backup.

**Don't let the trees keep you from seeing the forest!**

Reply:
> The problem I see there is that newer versions of the stock firmware behave much more annoying. You can no longer abort the wizard on first start, completing it requires a DHCP server connected to the WAN port, ...

No problem, and it doesn't even take 30 seconds to complete the wizard and whether you want to accept it or not, now 99% of the routers comes with signed firmware and the current guide is too difficult to understand, it is better that everyone has the same firmware "signed" and so you avoid problems and facilitate the understanding of the guide for everyone.

> If you happen to get a device with an old version of the stock firmware, you should be thankful and take the easy route of using that to flash the unsigned image.

In the old "unsigned" stock firmware the recovery mode does not work (hold down the "reset" button while turning on the device), but in the "signed" firmware the recovery mode works (I'm talking about stock firmware not OpenWrt) and maybe one day it can be useful for someone.

> There is no advantage what-so-ever of first upgrading and then using the signed installer image -- everything is going to be replaced anyway no matter if using signed or unsigned installer image, the end result is the same.

These changes to the guide will make it easier for anyone to understand, because right now it has too much useless and repetitive information and prevents you from being asked: "why I can't install the firmware" or "why I can't downgrade the firmware" or "why my router bricked when I was doing the full backup".

**Don't be afraid of change when it's for the better!**

**You can’t see the forest for the trees!**

Reply:
> The problem I see there is that newer versions of the stock firmware behave much more annoying. You can no longer abort the wizard on first start, completing it requires a DHCP server connected to the WAN port, ...

No problem, and it doesn't even take 60 seconds to complete the wizard and now 99% of the routers comes with "signed" firmware and the current guide is too difficult to understand, it is better that everyone has the same firmware "signed" and so you avoid problems and facilitate the understanding of the guide for everyone.

> If you happen to get a device with an old version of the stock firmware, you should be thankful and take the easy route of using that to flash the unsigned image.

In the old "unsigned" stock firmware the TFTP recovery mode does not work (hold down the "reset" button while turning on the device), but in the "signed" firmware the TFTP recovery mode works (I'm talking about stock firmware not OpenWrt firmware) and maybe one day it can be useful for someone.

> There is no advantage what-so-ever of first upgrading and then using the signed installer image -- everything is going to be replaced anyway no matter if using signed or unsigned installer image, the end result is the same.

These changes to the guide will make it easier for anyone to understand, because right now it has too much useless and repetitive information and prevents you from being asked: "why I can't install the firmware" or "why I can't downgrade the firmware" or "why my router bricked when I was doing the full backup".

**Don't be afraid of change when it's for better!**

**You can’t see the forest for the trees!**

**Please give it a chance.**

Reply:
> The problem I see there is that newer versions of the stock firmware behave much more annoying. You can no longer abort the wizard on first start, completing it requires a DHCP server connected to the WAN port, ...

It's not a real problem and it doesn't even take 60 seconds to complete the wizard and now 99% of the new routers comes with "signed" firmware and the current guide is too difficult to understand, it is better that everyone has the same firmware "signed" and to avoid problems and facilitate the understanding of the guide for everyone.

> If you happen to get a device with an old version of the stock firmware, you should be thankful and take the easy route of using that to flash the unsigned image.

In the old "unsigned" stock firmware the TFTP recovery mode does not work (hold down the "reset" button while turning on the device), but in the "signed" firmware the TFTP recovery mode works (I'm talking about stock firmware not OpenWrt firmware) and maybe one day it can be useful for someone.

> There is no advantage what-so-ever of first upgrading and then using the signed installer image -- everything is going to be replaced anyway no matter if using signed or unsigned installer image, the end result is the same.

These changes to the guide will make it easier for anyone to understand, because the current guide has too much useless and repetitive information and the new guide avoids you being asked: "why I can't install the firmware" or "why I can't downgrade the firmware" or "why my router bricked when I was doing the full backup".

**Don't be afraid of change when it's for better!**

**You can’t see the forest for the trees!**

**Please give it a chance.**